tests/utils.go

```diff
@@ -234,3 +237,45 @@ func findConditionByType(conditionType cdiv1.DataVolumeConditionType, conditions
 	}
 	return nil
 }
+
+// MakePrometheusHTTPRequest makes a request to the prometheus api and returns the response
+func MakePrometheusHTTPRequest(f *framework.Framework, endpoint string) *http.Response {
+	var token, host string
+	var err error
+	var resp *http.Response
+
+	gomega.Eventually(func() bool {
+		host, err = RunOcCommand(f, "-n", "openshift-monitoring", "get", "route", "prometheus-k8s", "--template", "{{.spec.host}}")
```

Comment:
If we use kubevirtci prometheus, the namespace will likely be different. Maybe investigate adding a prometheus-namespace to the function test suite, and use that to figure out which namespace to get information from.

Reply:
I already have that ns on the operator's env vars, and we could also check if openshift and adjust accordingly as fallback

but yeah mm this will require some changes to run on both, would prefer to tackle it later on and not in this PR, would that be ok?

Reply:
Yeah that is fine.